stream_alert/athena_partition_refresh/main.py

```diff
@@ -605,10 +632,13 @@ def handler(*_):
 
     # Check that the 'streamalert' database exists before running queries
     if not stream_alert_athena.check_database_exists():
-        raise AthenaPartitionRefreshError('The \'streamalert\' database does not exist')
+        raise AthenaPartitionRefreshError(
+            'The \'{}\' database does not exist'.format(stream_alert_athena.sa_database)
+        )
 
     if not stream_alert_athena.add_hive_partition(s3_buckets_and_keys):
-        raise AthenaPartitionRefreshError('Adding partition(s) has failed')
+        LOGGER.error('Failed to add hive partition(s)')
```

Comment:
Why don't you want to raise the exception here? Any concern doing that?

Reply:
So this was changed because the `add_hive_partition` can return `False` even if there wasn't real error that requires an `exception`ional response. Errors that should raise an exception now raise directly from within `add_hive_partition`